### PR TITLE
Slim

### PR DESCRIPTION
**Pull request by dgdavid**

### CODE REVIEW

File: app/views/diaries/edit.html.slim [hunk 1]
```diff
@@ -1,3 +1,3 @@
-%h1 New diary
+h1 Edit diary
```
Comment (DavidKang) [inline app/views/diaries/edit.html.slim]: Why do you rename this file??

File: app/views/diaries/new.html.slim [hunk 2]
```diff
@@ -1,3 +1,3 @@
-%h1 Edit diary
+h1 New diary
```
Comment (DavidKang) [inline app/views/diaries/new.html.slim]: The same with this

File: app/views/diaries/edit.html.slim (hunk 1 above)
Comment (dgdavid) [inline app/views/diaries/edit.html.slim]: Actually, I didn't rename it. But, I don't know why, Git believes that I renamed this from «new» to «edit» and vice versa.

**Comment by DavidKang** [comment]
+1

**Pull request merged**